elementary/gtk-3.0/gtk-widgets.css

```diff
@@ -4209,6 +4209,10 @@ dialog toolbar {
     background-color: @bg_color;
 }
 
+.dialog-action-area button > * {
+    min-height: calc(1em + 7px);
```

Comment:
Do we wanna comment how this was derived?

Reply:
tbh I don't really understand why this works. I guess it's like theoretically the alignment inside the button or grid or whatever it is should be the height of the text plus padding, but then for some reason it's off by one? So maybe there's some kind of rounding? I dunno, it's magic numbers

Reply:
Good enough for me 😜

Reply:
Could it be off by 1px because it's accounting for the text-shadow?